test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -13,15 +12,14 @@
 TEST_ROOT = os.path.dirname(__file__)
 GOOD_MANIFEST = os.path.join(TEST_ROOT, 'test_product.json_manifest')
 
-GETCAPABILITIES_FILE = os.path.join(TEST_ROOT, 'getCapabilities.xml')
 GETFEATURE_FILE = os.path.join(TEST_ROOT, 'getFeature.json')
-
-with open(GETCAPABILITIES_FILE) as f:
-    TEST_GETCAPABILITIES_RESPONSE = httpretty.Response(f.read())
+GETFEATURE_OLD_PRODUCTS_FILE = os.path.join(TEST_ROOT, 'getFeature_old_products.json')
 
 with open(GETFEATURE_FILE) as f:
     TEST_GETFEATURE_JSON = f.read()
-TEST_GETFEATURE_RESPONSE = httpretty.Response(TEST_GETFEATURE_JSON)
+
+with open(GETFEATURE_OLD_PRODUCTS_FILE) as f:
+    TEST_GETFEATURE_OLD_PRODUCTS_JSON = f.read()
```

Comment:
Because the handler is now doing TWO getfeature requests, we need to mock this second one as well. How do I do that?

Reply:
Use `side_effect` rather than return value. if you set side_effect to an Iterable, it will return the elements in sequence when the mock is called repeatedly:

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.side_effect